EdFi.Buzz.Database/migrations/sample-data/20201022195215-adds-sample-data.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var fs = require('fs');
+var path = require('path');
+var Promise;
```

Comment:
This is surprising to me. Do we need a Promise variable? I would have thought the Promise object was enough.

Reply:
This is the file generated by db-migrate. Once generated I didn't change anything. So, I guess the answer is Yes. hehe....